crates/wasm-shrink/src/lib.rs

```diff
@@ -216,33 +217,24 @@ impl ShrinkRun {
     }
 
     fn validate_wasm(&self, wasm: &[u8]) -> Result<()> {
-        let mut validator = wasmparser::Validator::new_with_features(wasmparser::WasmFeatures {
-            // TODO: we should have CLI flags for each Wasm proposal.
-            reference_types: true,
-            multi_value: true,
-            bulk_memory: true,
-            simd: true,
-            threads: true,
-            shared_everything_threads: false,
-            tail_call: true,
-            multi_memory: true,
-            exceptions: true,
-            memory64: true,
-            relaxed_simd: true,
-            extended_const: true,
-            mutable_global: true,
-            saturating_float_to_int: true,
-            sign_extension: true,
-            component_model: false,
-            function_references: false,
-            gc: false,
-            custom_page_sizes: false,
-            component_model_values: false,
-            component_model_nested_names: false,
-
-            floats: true,
-            memory_control: true,
-        });
+        let mut validator = wasmparser::Validator::new_with_features(
+            WasmFeatures::REFERENCE_TYPES
+                | WasmFeatures::MULTI_VALUE
+                | WasmFeatures::BULK_MEMORY
+                | WasmFeatures::SIMD
+                | WasmFeatures::THREADS
+                | WasmFeatures::TAIL_CALL
+                | WasmFeatures::MULTI_MEMORY
+                | WasmFeatures::EXCEPTIONS
+                | WasmFeatures::MEMORY64
+                | WasmFeatures::RELAXED_SIMD
+                | WasmFeatures::EXTENDED_CONST
+                | WasmFeatures::MUTABLE_GLOBAL
+                | WasmFeatures::SATURATING_FLOAT_TO_INT
+                | WasmFeatures::SIGN_EXTENSION
+                | WasmFeatures::FLOATS
+                | WasmFeatures::MEMORY_CONTROL,
+        );
```

Comment:
Mind replacing this with `::all()` since that's now available?

Reply:
we can but then we'd need to disable 7 of the flags again. Not sure if that's more readable.